src/pytest_postgresql/factories.py

```diff
@@ -213,14 +213,17 @@ def postgresql_noproc_fixture(request: FixtureRequest) -> NoopExecutor:
 
 
 def postgresql(
-        process_fixture_name: str, db_name: str = None, load: List[str] = None
+        process_fixture_name: str, db_name: str = None, load: List[str] = None,
+        isolation_level: int = 0,
 ) -> Callable[[FixtureRequest], connection]:
     """
     Return connection fixture factory for PostgreSQL.
 
     :param process_fixture_name: name of the process fixture
     :param db_name: database name
     :param load: SQL to automatically load into our test database
+    :param isolation_level: optional postgresql isolation level
```

Comment:
does it makes sense to also be able to configure it with pytest options?

Reply:
I'm not sure this would make sense to set via pytest... for my use case there is only one test where I need to alter the value.  The rest run fine without this being set.